src/ornament.ts

```diff
@@ -286,21 +314,21 @@ export class Ornament extends Modifier {
 
     if (this.accidentalLower) {
       this.accidentalLower.render(ctx, glyphX, glyphY);
-      glyphY -= this.accidentalLower.getMetrics().height;
+      glyphY -= this.accidentalLower.getMetrics().height ?? 0;
```

Comment:
Same, silent failure here.

Reply:
Resolved in glyph calculating height and width